src/command.rs

```diff
@@ -341,6 +351,38 @@ impl App {
                 m.inline_defs();
                 println!("{}", printer::fmt(&m));
             }
+            Command::BoundedCheck { depth, .. } => {
+                let mut universe = HashMap::new();
+                let stdin = std::io::stdin();
+                for sort in &m.signature.sorts {
+                    println!("how large should {} be?", sort);
```

Comment:
Eventually I'd like this to be available as command line arguments but this interactive input is fine for an initial PR.

Reply:
I was envisioning something like `--sort1 2 --sort2 4 --depth 5` but I wasn't sure how to make that work with clap.